src/quo2/components/notifications/activity_log/style.cljs

```diff
@@ -25,11 +26,12 @@
 (def message-body
   {:color colors/white})
 
-(def message-container
+(defn message-container
+  [content-type]
   {:border-radius      12
    :margin-top         12
    :padding-horizontal 12
-   :padding-vertical   8
+   :padding-vertical   (if (= content-type constants/content-type-image) 12 8)
```

Comment:
Not sure about this condition's flexibility for other content types. 

I think we have only two variants `8` (used on text) and `12` (images, stickers). We can use a boolean to toggle between them depending on the type.  Not for this PR. We will need this once we update the designs for other content types.

Reply:
Hi @smohamedjavid,
Thank you very much for your input. Updated the condition as per your suggestion. And instead of passing content-type, passed :attachment prop as per figma.

@ilmotta not sure if there is an issue, but we properly should create [Activity logs attachments](https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=13336-165824&mode=design&t=CmCRjqjhp4xfwVGF-0) and refactor and simplify activity logs component

Reply:
> not sure if there is an issue, but we probably should create [Activity logs attachments](https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=13336-165824&mode=design&t=CmCRjqjhp4xfwVGF-0) component and refactor and also simplify activity logs component. wdyt?

@Parveshdhull, I totally agree, we need to catch up to the latest changes in the DS and the activity log component is a bit outdated too (based on the guidelines).